service/ziti-tunnel/service/state.go

```diff
@@ -154,12 +155,15 @@ func (t *RuntimeState) ToMetrics() dto.TunnelStatus {
 	for _, id := range t.ids {
 		AddMetrics(id)
 		clean.Identities[i] = &dto.Identity{
-			Name:        id.Name,
-			FingerPrint: id.FingerPrint,
-			Metrics:     id.Metrics,
-			Active:      id.Active,
-			MfaEnabled:  id.MfaEnabled,
-			MfaNeeded:   id.MfaNeeded,
+			Name:            id.Name,
+			FingerPrint:     id.FingerPrint,
+			Metrics:         id.Metrics,
+			Active:          id.Active,
+			MfaEnabled:      id.MfaEnabled,
+			MfaNeeded:       id.MfaNeeded,
+			MinTimeout:      id.MinTimeout,
```

Comment:
all these timout/times should probably be put into a mfa struct or at least have 'mfa' added to their names to make it clear what they're for